cmd/experimental/kjobctl/pkg/constants/constants.go

```diff
@@ -18,6 +18,7 @@ package constants
 
 const (
 	ProfileLabel = "kjobctl.x-k8s.io/profile"
+	ModeLabel    = "kjobctl.x-k8s.io/mode"
```

Comment:
Can we just have this for slurm related jobs and therefore reduce the impact on the other kjob modes?

Reply:
I'm thinking of setting the label only when it's a slurm job, otherwise we consider it a standalone job.

This way we don't need to do anything for standalone jobs, Ray modes and so on.

Reply:
But what if in the future we add more modes that will overlap with other types of tasks?

Reply:
I also can see that this changes already merged https://github.com/kubernetes-sigs/kueue/blob/main/cmd/experimental/kjobctl/pkg/constants/constants.go#L21 :) I think I need to rebase it.

Reply:
It's very strange that git didn't show any conflicts. But maybe it's just because the changes are the same.